ui/admin/app/components/form/storage-bucket/aws/index.hbs

```diff
@@ -221,9 +221,9 @@
     {{#each this.credentials as |credentialType|}}
       <G.RadioCard
         @value={{credentialType}}
-        @checked={{eq credentialType this.selectedCredentialType}}
+        @checked={{eq credentialType @model.credentialType}}
         @disabled={{form.disabled}}
-        {{on 'input' (fn this.updateCredentialTypeSelection credentialType)}}
+        {{on 'input' (fn @changeCredentialType credentialType)}}
```

Comment:
Neet! I forgot we have `changeCredentialType` within the controller, nice clean up of the component deleting `updateCredentialTypeSelection`!!